examples/cms-agilitycms/pages/api/generatePreviewKey.js

```diff
@@ -0,0 +1,12 @@
+import { generatePreviewKey } from "@agility/nextjs/node"
+
+export default async (req, res) => {
+
+	//TODO: Only generate the preview link if you are already in Preview!
```

Comment:
TODO, also formatting 👍

Reply:
created an issue so we have this todo tracked: https://github.com/agility/agilitycms-nextjs-starter/issues/24